src/main/java/com/parasoft/findings/utils/common/util/XMLUtil.java

```diff
@@ -63,34 +66,70 @@ public static SAXParserFactory getSaxParserFactory() {
         return _saxParserFactory;
     }
 
+    static boolean shouldDisallowDTD()
```

Comment:
This method always returns true(we don't need to use system property), consider removing it and related code.